### PR TITLE
removed expected working time from show/create/index of exercises

### PR DESCRIPTION
**Pull request by ThommyH**
we do not use it

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -47,6 +47,13 @@
     t.string   "oauth_secret", limit: 255
   end
 
+  create_table "copy_paste_events", id: false, force: :cascade do |t|
```
Comment (Parnswir) [inline db/schema.rb]: Most of the schema changes are unrelated to the working time.

**Comment by ThommyH** [comment]
removed them
